### PR TITLE
chore: update devDeps of @kubernetes/client-node needed for prepare lifecycle hook to match the commit hash we're pulling

### PR DESCRIPTION
**Pull request by gregmagolan**
Last one

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,11 +25,10 @@
...
             "@kubernetes/client-node": {
                 "//1": "these deps are from devDependencies of @kubernetes/client-node and needed for 'prepare' lifecycle hook",
-                "//2": "https://github.com/kubernetes-client/javascript/blob/763decf6e09d0e246b80cefdb2c5fd570373801a/package.json#L72",
+                "//2": "https://github.com/kubernetes-client/javascript/blob/fc681991e61c6808dd26012a2331f83671a11218/package.json",
                 "dependencies": {
-                    "@types/node": "^10.12.0",
-                    "@types/underscore": "^1.8.9",
-                    "typescript": "~4.7.4"
+                    "@types/underscore": "^1.11.4",
+                    "typescript": "~4.9.4"
```
Comment (jbedard) [inline package.json:31]: Why these ones?

**Review by jbedard — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (gregmagolan) [reply package.json:31]: comment explains why,
```
            "@kubernetes/client-node": {
                "//1": "these deps are from devDependencies of @kubernetes/client-node and needed for 'prepare' lifecycle hook",
                "//2": "https://github.com/kubernetes-client/javascript/blob/fc681991e61c6808dd26012a2331f83671a11218/package.json",
                "dependencies": {
                    "@types/underscore": "^1.11.4",
                    "typescript": "~4.9.4"
                }
            }
```

TL;DR is they are needed for the prepare lifecycle hook but they are specified only as devDependencies in the package.json of the package

Comment (jbedard) [reply package.json:31]: Oh right, I was looking at that 10 minutes ago 😅

(and on mobile right now the diff looks ugly and it's hard to see 🫣)
